aboutus/index.php

```diff
@@ -771,10 +771,10 @@
                     <div id="form-div">
                         <form action="index.php" method="post" class="form" id="form1">
                             <p class="name">
-                                <input name="name" type="text" class="validate[required,custom[onlyLetter],length[0,100]] feedback-input" placeholder="Name" id="name" />
+                                <input name="name" type="text" class="validate[required,custom[onlyLetter],length[0,100]] feedback-input" placeholder="       Name" id="name" />
```

Comment:
* This is a hackish way to go about it. 
* We care about code as much as we care about design. 
* Use CSS to your aid.